doc/release-notes/rl-2511.section.md

```diff
@@ -49,6 +49,9 @@
 - `reaction` has been updated to version 2, which includes some breaking changes.
   For more information, [check the release article](https://blog.ppom.me/en-reaction-v2).
 
+- `searx` was updated to use `envsubst` instead of `sed` for parsing config environment variables.
```

Comment:
What do we need to migrate? To me it looks like things stay compatible.

Reply:
It a breaking change: `@SECRET@` → `$SECRET`.

Reply:
This could be explained directly here, though. With a couple of examples.

Reply:
That definitely needs to be explained like this. I didn't even grasp it when first time reading it.

Reply:
I added one example of @SECRET@ to $SECRET. If you think we need more I can add another.